pkg/reconciler/common/tektoninstallerset/types.go

```diff
@@ -0,0 +1,65 @@
+/*
+Copyright 2022 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tektoninstallerset
+
+import (
+	"context"
+	"fmt"
+
+	mf "github.com/manifestival/manifestival"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type ComponentInstaller interface {
+	GetManifest(ctx context.Context) (*mf.Manifest, error)
```

Comment:
Any reason for passing `context.Context` on those ? They seem to be never used.

Reply:
in the defaultImplementation no. However it is an extension point for the future. So that in the future ctx could be used in other implementations without having to change the signature of methods in this interface.

Reply:
for example: platfrom contextKey from ctx.

Reply:
Ok 👌🏼 (for me, no need for now, so I would just make the interface evolve when we need it, but that's a nit 😝 )